### ISSUE TITLE
Option to quickly look up glossary terms from within game's rules (by clicking on them, for instance)

### ISSUE THREAD

**Issue by jan-kleks** [KyleU/solitaire.gg#43]
[Example](http://www.goodsol.com/pgshelp/index.html?klondike.htm). Yeah, blue hyperlinks might be a bit distracting, but you get a general idea. Check also these nice "how to play" examples with graphical card symbols if you click on, e.g. "down by alternate color".

**Comment by KyleU** [comment]
Agreed, I'll wire up the engine today. The annoyingly hard part will be actually providing non-plagiarized definitions of the terms in a localizable way.

**Comment by jan-kleks** [comment]
Here you go: https://en.wikipedia.org/wiki/Glossary_of_patience_terms
